ui/app/AppLayouts/Profile/popups/ChangePasswordModal.qml

```diff
@@ -60,7 +60,7 @@ StatusModal {
         }
     }
 
-    width: 480
+    width: 650
```

Comment:
Following design shared by Ben, it should be kept as `480`!

Reply:
Agreed, this was just a proposal. We do have agreement